src/main/java/io/jenkins/plugins/zscaler/SCMDetails.java

```diff
@@ -37,4 +40,27 @@ public static void populateSCMDetails(EnvVars env, BuildDetails buildDetails){
       buildDetails.setCommitSha(env.get(SCMConstants.SvnRevision));
     }
   }
+
+  /**
+   * Masks access token in git URL if present
+   *
+   * @param repoUrl
+   * @return
+   */
+  private static String maskAccessToken(String repoUrl) {
+    /*
+    If the repo URl is either of the following formats:
+    https://<token>@github/<username>/<repo>
+    or
+    https://oauth2:<token>@gitlab/<groupname>/<repo>
+    then, find the index of '@' and first '/'.
+    Then replace the string between 'https://' and '@' with mask string
+     */
+    int index = repoUrl.indexOf("@");
+    if (index > 0) {
+      int firstSlashIndex = repoUrl.indexOf("/");
```

Comment:
Can we use URL methods to extract domain and path and contruct back?

Reply:
Sure @jragula-zs